.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would rather not have any editor specific config files checkout in the source code. Could you remove this ?

Reply:
Yes, I have to add .idea/ to the gitignore.

Reply:
A suggestion for you is to have folders like that in your personal gitignore ($HOME/.gitignore)